workspaces/adventure-pack/goodies/python3/src/traverse_preorder/goody.json

```diff
@@ -0,0 +1,3 @@
+{
+  "name": "TraversePreorder"
```

Comment:
Since adding this goody gives you a `traverse_preorder` function, let's name the goody `traverse_preorder` in the same way?

Reply:
Done.